tests/yarn/model_llama2_7b_vllm.py

```diff
@@ -0,0 +1,53 @@
+import os.path
+from typing import List
+
+from base_model import BaseModel
+from vllm import LLM, SamplingParams
+from verification_prompt import PROMPT
+
+# MODEL_ID = 'Llama2/Llama-2-7B-fp16'
+MODEL_ID = "NousResearch/Yarn-Llama-2-7b-64k"
+
+
+class Model(BaseModel):
+
+    def __init__(self):
+        super().__init__()
+        self.model_id = MODEL_ID
+        self.llm = LLM(
+            model=MODEL_ID,
+            max_model_len=8192,
+            max_num_batched_tokens=8192,
+            seed=42,
+        )
```

Comment:
Add this parameter:
`trust_remote_code=True,`

It is required to bypass the checks in the transformers library.